pyproject.toml

```diff
@@ -76,5 +76,5 @@ celery = ["celery", "croniter"]
 immuni-common-dev = { path = "dev", develop = true}
 
 [build-system]
-requires = ["poetry>=0.12"]
+requires = ["poetry>=0.12", "setuptools"]
```

Comment:
Isn't this merged into develop already?
Also, it seems that the latest poetry now fixed the issue. I'd see whether I can fix it in a separate PR.

Reply:
yes now is completely merged